databaker/bake.py

```diff
@@ -206,30 +216,33 @@ def value_for_dimension(dimension):
         values = {}
         values[OBS] = obj.value
 
-        for dimension in range(DATAMARKER, LAST_METADATA + 1):
+        for dimension in range(OBS+1, LAST_METADATA + 1):
             values[dimension] = value_for_dimension(dimension)
 
         # Mutate values
         # Special handling per dimension.
-
-        if not isinstance(values[OBS], float):  # NOTE xls specific!
-            ob_value, dm_value = parse_ob(ob)
-            values[OBS] = ob_value
-            # the observation is not actually a number
-            # store it as a datamarker and nuke the observation field
-            if values[DATAMARKER] == '':
-                values[DATAMARKER] = dm_value
-            elif dm_value:
-                logging.warn("datamarker lost: {} on {!r}".format(dm_value, ob))
-
-        if values[TIMEUNIT] == '' and values[TIME] != '':
-            # we've not actually been given a timeunit, but we have a time
-            # determine the timeunit from the time
-            values[TIMEUNIT] = datematch(values[TIME])
+        # NOTE  - varibales beinging SH_ ... are ependant on user choices from the bottom of the template file
+
+        if SH_Split_OBS:
+            if not isinstance(values[OBS], float):  # NOTE xls specific!
+                ob_value, dm_value = parse_ob(ob)
+                values[OBS] = ob_value
+                # the observation is not actually a number
+                # store it as a datamarker and nuke the observation field
+                if values[OBS+1] == '':
```

Comment:
Not convinced these should be modifying "next value after `OBS`"; we actually want to modify the `DATAMARKER` field. Unlike L219 where we actually mean "next value after `OBS`".